packages/components/pie-modal/src/modal.scss

```diff
@@ -53,4 +53,10 @@
         padding-block: var(--modal-content-padding-block-start) var(--modal-content-padding);
         padding-inline: var(--modal-content-padding);
     }
+
+    & .c-modal-closeBtn {
+        position: absolute;
+        right: var(--dt-spacing-d);
+        top: var(--dt-spacing-d);
```

Comment:
```suggestion
        inset-inline-end: var(--dt-spacing-d);
        inset-block-start: var(--dt-spacing-d);
```

Reply:
Could I add this as part of my next pr please :)